markdown/blog/2025-gsoc-phase-1.md

```diff
@@ -0,0 +1,179 @@
+---
+title: "2025 Google Summer of Code: Phase 1"
+date: 2025-05-27T06:00:00+01:00
+type: Community
+tags:
+  - Mentorship
+  - GSoC
+  - AsyncAPI
+  - OSS
+cover: /img/posts/gsoc-25.webp
+authors:
+  - name: Azeez Elegbede
+    photo: /img/avatars/ace.webp
+    link: https://www.linkedin.com/in/aelegbede/
+    byline: AsyncAPI Preacher 
+---
+
+**At AsyncAPI, Summer means one thing: Google Summer of Code**
+
+Every year, we eagerly anticipate this season, not just because of the program itself, but because of what it represents for our community. GSoC isn't just about the stipend contributors receive for their participation. For us, it's about the doors it opens, the chance to welcome new developers, foster fresh perspectives, and nurture contributors who often go on to become long-term contributors of the project. 
```

Comment:
```suggestion
Every year, we eagerly anticipate this season, not just because of the program itself, but because of what it represents for our community. GSoC isn't just about the stipend contributors receive for their participation. For us, it's about the doors it opens, the chance to welcome new developers, foster fresh perspectives, and nurture contributors who often go on to become long-term contributors to the project. 
```